Question packs

mezotv opened the pull request.

mezotv commented:
Could you attach a video of how it looks like on your machine?

Honey2339 commented:
This is how it looks on my machine

https://github.com/Would-You-Bot/website/assets/121949298/b8ca2371-5e0c-42b2-866e-4db9143d7713

mezotv reviewed (changes requested):
Looks really good however could you make a little transition so it looks a bit smother?

Honey2339 commented:
Yeah sure working on it.

Honey2339 commented:
Oh you mean that it should not copy "/debug *<cmd>*"? right?

mezotv commented:
> Oh you mean that it should not copy /debug "cmd"? right?

Thats fine but it should only copy /debug and not the <subcommand> part of it.

Honey2339 commented:
I have implemented the copying of only command part but im finding it hard to implement a transition between the text.

mezotv commented:
> I have implemented the copying of only command part but im finding it hard to implement a transition between the text.

Thats fine! Ill check in a bit.

Podskio commented:
> I have implemented the copying of only command part but im finding it hard to implement a transition between the text.

Try splitting the whitespace, eg `"/debug <subcommand>".split(" ")[0]`

taqh committed:
feat: added create packs form

mezotv committed:
fix: add suggested changes

mezotv committed:
fix: capitalize types

dpaulos6 committed:
feat: add text glow + focus state outline

taqh committed:
feat: persist entered pack data between steps

taqh committed:
fix: sync tags in localstorage with user changes

taqh committed:
feat: built packs page

taqh committed:
updated styling

dpaulos6 committed:
feat: update dialog width

dpaulos6 committed:
feat: Packs page (#60)

dpaulos6 committed:
feat: prisma + mongodb

Also included some vscode extension recommendations related to prisma support, but also eslint and prettier.

dpaulos6 committed:
Merge branch 'main' into question-packs
